src/runtime_src/driver/xclng/tools/xbutil/xbutil.cpp

```diff
@@ -746,7 +745,7 @@ std::unique_ptr<xcldev::device> xcldev::xclGetDevice(unsigned index)
             std::cout << "ERROR: Card index " << index << " out of range";
             std::cout << std::endl;
         } else {
-          return std::make_unique<xcldev::device>(index,nullptr);
+         return std::make_unique<xcldev::device>(index,nullptr);
```

Comment:
Sorry that I was not clear before. I meant that the indention should be 4 spaces as the rest of the code does. Or maybe it was just showing strangely in my browser?

Reply:
Should it be aligned with the try-catch statements? I thought that since it was inside the else statement, it should be aligned with line 746